src/options/views/edit/settings.vue

```diff
@@ -129,35 +97,56 @@ export default {
       }
     },
   },
+  methods: {
+    countRows(name) {
+      return 2 + (this.settings.custom?.[name]?.match(/\n/g)?.length || 0);
```

Comment:
This does not work on long lines that wrap. Maybe it's better to use `scrollHeight` in a debounced callback? Or just leave it as is.

Reply:
Fixed.